torchvision/ops/boxes.py

```diff
@@ -133,6 +134,54 @@ def clip_boxes_to_image(boxes: Tensor, size: Tuple[int, int]) -> Tensor:
     return clipped_boxes.reshape(boxes.shape)
 
 
+def box_convert(boxes: Tensor, in_fmt: str, out_fmt: str) -> Tensor:
+    """
+    Converts boxes from given in_fmt to out_fmt.
+
+    Arguments:
+        boxes (Tensor[N, 4]): boxes which will be converted.
+        in_fmt (str): Input format of given boxes.
+        out_fmt (str): Output format of given boxes.
```

Comment:
Same here, can you also specify the list of supported formats? Like `supported formats are ['xyxy', 'xywh', 'cxcywh']`

Reply:
Added 👍